### PR TITLE
Added ability to use tuned model with ChatVertexAI

### PR DESCRIPTION
**Pull request by arianjahiri**
- **Description:** Was not able to use a tuned model when using "ChatVertexAI" added that based on how it was done in non chat VertexAI class.
  - **Issue:** N/A
  - **Dependencies:** N/A
  - **Tag maintainer:** @baskaryan 
  - **Twitter handle:** @arian_jahiri

### CODE REVIEW

**Comment by arianjahiri** [comment]
example of usage of ChatVertexAI with a tuned model:
`chat = ChatVertexAI(max_output_tokens= 1024, tuned_model_name="GOOGLE_TUNED_MODEL_PATH")`

File: libs/langchain/langchain/chat_models/vertexai.py [hunk 1]
```diff
@@ -122,6 +122,8 @@ class ChatVertexAI(_VertexAICommon, BaseChatModel):
     "Underlying model name."
     examples: Optional[List[BaseMessage]] = None
 
+    tuned_model_name: Optional[str] = None
```
Comment (baskaryan) [inline libs/langchain/langchain/chat_models/vertexai.py:125]: is there any way to identify tuned_model_name from the name alone? wondering if we need separate params for model_name vs tuned_model_name

Comment (baskaryan) [reply libs/langchain/langchain/chat_models/vertexai.py:125]: cc @lkuligin

Comment (arianjahiri) [reply libs/langchain/langchain/chat_models/vertexai.py:125]: I guess you could check if the "model_name" is in this list of tuned models:

**Comment by arianjahiri** [comment]
@baskaryan Any update on this?

File: libs/langchain/langchain/chat_models/vertexai.py (hunk 1 above)
Comment (vtenneti-wiq) [reply libs/langchain/langchain/chat_models/vertexai.py:126]: Hello when I try to use the  tuned_model_name parameter, it is ignored. When I look at the latest langchain source I see ChatVertexAI actually doesn't use this at all.
Was this change not merged in at all?
How do I use a tuned model with ChatVertexAI please?

**Pull request closed without merging**